src/vuln_analysis/server/webhooks/config.py

```
@@ -109,8 +109,8 @@ class WebhookConfig:
     def __post_init__(self):
         if self.max_attempts < 1:
             raise ValueError(f"{_ENV_PREFIX}MAX_ATTEMPTS must be at least 1")
-        if self.max_rounds < 0:
-            raise ValueError(f"{_ENV_PREFIX}MAX_ROUNDS must be non-negative")
+        if self.max_rounds < 1:
+            raise ValueError(f"{_ENV_PREFIX}MAX_ROUNDS must be at least 1")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)